src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -1,5 +1,6 @@
 /**
- * (C) Copyright 2022-2024 Intel Corporation.
+ * (C) Copyright 2022-2025 Intel Corporation.
```

Comment:
you should revert the intel copyright update

Reply:
All CI tests passed in build 3 without issue. The copyright was just fixed. "Doc-only: true" was set to avoid tests.